### PR TITLE
update google protobuf to 3.21.10

### PR DESCRIPTION
**Pull request by myk002**

### CODE REVIEW

File: depends/protobuf/src/google/protobuf/parse_context.h [hunk 1]
```diff
@@ -946,19 +946,19 @@ PROTOBUF_NODISCARD const char* WireFormatParser(T& field_parser,
...
     void* object, const char* ptr, ParseContext* ctx);
-PROTOBUF_NODISCARD PROTOBUF_EXPORT const char* PackedSInt32Parser(
+PROTOBUF_NODISCARD /*PROTOBUF_EXPORT*/ const char* PackedSInt32Parser(
     void* object, const char* ptr, ParseContext* ctx);
-PROTOBUF_NODISCARD PROTOBUF_EXPORT const char* PackedSInt64Parser(
+PROTOBUF_NODISCARD /*PROTOBUF_EXPORT*/ const char* PackedSInt64Parser(
     void* object, const char* ptr, ParseContext* ctx);
-PROTOBUF_NODISCARD PROTOBUF_EXPORT const char* PackedEnumParser(
+PROTOBUF_NODISCARD /*PROTOBUF_EXPORT*/ const char* PackedEnumParser(
     void* object, const char* ptr, ParseContext* ctx);
```
Comment (ab9rf) [inline depends/protobuf/src/google/protobuf/parse_context.h:963]: sadly these changes are not enough by themselves to get this to compile in MSVC v140

Comment (myk002) [reply depends/protobuf/src/google/protobuf/parse_context.h:963]: Will that matter once we move to the new toolchain?

Comment (ab9rf) [reply depends/protobuf/src/google/protobuf/parse_context.h:963]: i'm hoping not but i have not tested yet, too many other things to do atm

**Comment by myk002** [comment]
closing in favor of #2485

**Pull request closed without merging**